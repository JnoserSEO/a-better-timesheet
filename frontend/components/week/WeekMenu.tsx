import React from 'react'
import TimesheetView from '../../models/view.model'
import WeekStyleSwitch from './WeekStyleSwitch'
import '../../styles/week/week-menu.scss'

interface WeekMenuProps {
    view:TimesheetView;
    callback:(view:TimesheetView)=>void;
}

const WeekMenu:React.FC<WeekMenuProps> =({view,callback})=>{
    return <div className="week-menu">
        <img height="51px" src="./img/calendar.svg" />
        <div className="week-menu-week-selector">
            <div className="week-not-selected">Last Week</div>
            <div className="week-selected">This Week</div>
            <div className="week-not-selected">Next Week</div>
        </div>
        <WeekStyleSwitch view={view} callback={callback} />
    </div>
}

export default WeekMenu