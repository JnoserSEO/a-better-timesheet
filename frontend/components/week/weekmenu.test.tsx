import React from 'react'
import { mount } from 'enzyme'
import WeekMenu from './WeekMenu'


test('should render WeekMenu', () => {
    const wrapper = mount(<WeekMenu view={'5 day'} dispatch={(x) => x} />)
    expect(wrapper.find).toMatchSnapshot()
})