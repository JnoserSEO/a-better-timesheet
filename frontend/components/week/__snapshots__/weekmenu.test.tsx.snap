// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render WeekMenu 1`] = `[Function]`;
